PetrolEngine.cpp

```diff
@@ -2,19 +2,36 @@
 #include <iostream>
 
 PetrolEngine::PetrolEngine(int power, float capacity, int gears)
-    : power_(power)
-    , capacity_(capacity)
-    , gears_(gears)
-    , currentGear_(0)
-{
+    : power_(power), capacity_(capacity), gears_(gears), currentGear_(0) {
     std::cout << __FUNCTION__ << std::endl;
 }
 
-void PetrolEngine::changeGear(int gear)
-{
-    // TODO: Add checking if gear is between -1 and gears_
-    // -1 is for REAR
-    // 0 is for NEUTRAL
+PetrolEngine::~PetrolEngine() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void PetrolEngine::changeGear(int gear) {
+    if (gear < -1 || gear > gears_) {
```

Comment:
O think that You should avoid magic numbers. Let's make - 1 as constexpr or const variable

Reply:
+1